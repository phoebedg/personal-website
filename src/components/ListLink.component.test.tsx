import React from "react";
import { render } from "@testing-library/react";
import { ListLink } from "./ListLink.component";

test("renders ListLink", async () => {
  const { container, getByText } = render(
    <ListLink
      title={"Personal Website"}
      href={"https://github.com/phoebedg/personal-website"}
    />
  );
  expect(getByText("Personal Website")).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
