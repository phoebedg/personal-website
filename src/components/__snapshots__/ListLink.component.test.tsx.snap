// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ListLink 1`] = `
<div>
  <a
    class="link"
    href="https://github.com/phoebedg/personal-website"
    rel="noopener noreferrer"
    target="_blank"
  >
    Personal Website
  </a>
</div>
`;
